code/index.html

```diff
@@ -1,35 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Clue</title>
-  </head>
-  <body>
-    <section class="cards">
-      <div class="killer-deck" id="killer" onclick="pickKiller()">
-        <p>KILLER</p>
-        <p id="killerName"></p>
-        <p id="killerAge"></p>
-        <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
-        <p id="killerDescription"></p>
-      </div>
-      <div class="weapon-deck" id="weapon" onclick="pickWeapon()">
-        <p>WEAPON</p>
-        <p id="weaponName"></p>
-        <p id="weaponWeight"></p>
-      </div>
-      <div class="room-deck" id="room" onclick="pickRoom()">
-        <p>ROOM</p>
-        <p id="roomName"></p>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Clue</title>
+</head>
+
+<body>
+  <header>
+    <img src="assets/gameleader2.jpeg" height="180" />
+    <p>Welcome to a game of Cluedo. Please pick details below.</p>
+  </header>
+  <section class="cards">
+    <div class="killer-deck" id="killer" onclick="pickKiller()">
+      <p class="cardName">KILLER?</p>
+      <img id="killerImage" src="assets/killer.jpg" height="180" />
+      <p id="killerName"></p>
+      <p id="killerAge"></p>
+      <p id="killerOccupation"></p>
+      <p id="killerDescription"></p>
+    </div>
+    <div class="weapon-deck" id="weapon" onclick="pickWeapon()">
+      <p class="cardName">WEAPON?</p>
+      <img id="weaponImage" src="assets/weapon.png" height="180" />
+      <p id="weaponName"></p>
+      <p id="weaponWeight"></p>
+      <p id="weaponColor"></p>
+    </div>
+    <div class="room-deck" id="room" onclick="pickRoom()">
+      <p class="cardName">ROOM?</p>
+      <img id="roomImage" src="assets/room.png" height="180" />
+      <p id="roomName"></p>
+    </div>
+    <div class="time-deck" id="time" onclick="pickTime()">
+      <p class="cardName">TIME?</p>
+      <img id="timeImage" src="assets/time.png" height="180" />
+      <p id="timeOfDay"></p>
+    </div>
+  </section>
+  <h1 id="mystery"></h1>
+  <button class="reveal" onclick="revealMystery()">REVEAL THE CRIME</button>
+  <h1 id="playAgain"></h1>
+  <button class="reload" onclick="refresh()">NEW GAME</button>
```

Comment:
Maybe this button doesn't needs to be here all the time? Only when you have revealed the crime